osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -124,7 +120,8 @@ public BufferedWedgeInfo(WorkingBeatmap beatmap)
                     }));
 
                     //get statistics from the current ruleset.
-                    labels.AddRange(beatmapInfo.Ruleset.CreateInstance().GetBeatmapStatistics(beatmap).Select(s => new InfoLabel(s)));
+                    if (beatmapInfo.Ruleset != null)

```

Comment:
I'd prefer if the default beatmap had a ruleset (potentially also dummy with empty statistics).